ui/components/MesheryApplications.js

```diff
@@ -648,7 +648,10 @@ function MesheryApplications({
                 title="click to download"
                 onClick={() => handleAppDownload(rowData.id ,rowData.type.String, rowData.name)}
               >
-                <img src={`/static/img/${(rowData.type.String).replaceAll(" ", "_").toLowerCase()}.svg`} width="45px" height="45px" />
+                {source_type== undefined || source_type == " " ?
```

Comment:
Uh Oh!  @legendarykamal This is showing that `source_type` is not defined. I'm now wondering how it worked for you locally. Did you test your changes?
![image](https://user-images.githubusercontent.com/65453826/194724089-e6cfb71e-0ffd-4f8a-be02-d86039759978.png)

Reply:
I see 👀 actually i had tested for grid view(ss is from it) and not table.
 This arises from the tabel view I will fix this right away .

Reply:
My bad I had seen and was gonna fix it earlier itself 😓 .